src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -529,7 +558,7 @@ public void onClose(int code,
 
         public void waitForInputStreamToConnect() {
             try {
-                this.openLatch.await();
+                this.openLatch.await(10, TimeUnit.SECONDS);
```

Comment:
What is the reasoning behind this change?

Reply:
In some cases the thread got waiting for ever.

Reply:
Should we leave this configurable then?